content/docs/setup/kubernetes/platform-setup/minikube/index.md

```diff
@@ -21,11 +21,18 @@ Follow these instructions to prepare Minikube for Istio.
     On Kubernetes **1.9**:
 
     {{< text bash >}}
-    $ minikube start --memory=4096 --kubernetes-version=v1.9.4 --vm-driver=`your_vm_driver_choice`
+    $ minikube start --memory=8192 --cpus=4 --kubernetes-version=v1.9.4 \
+        --extra-config=controller-manager.cluster-signing-cert-file="/var/lib/localkube/certs/ca.crt" \
```

Comment:
I am pretty sure the cluster-signing-cert-file is not needed - as minikube + Istio run without them in this exact same setup.  Could you try without them - as the command is a little clunky with them?  Otherwise looks good.

Reply:
Then why do we have it in circleci?

The idea is we recommend exactely what we test. Until we have the equivalent removed from the circleci file, we should have this here.

Reply:
Agree that circleci should test what we recommend.   If we think the settings are too high (e.g., 8GB) then another approach is to reduce the minikube memory used by circleci.

Reply:
Checked with @ayj and we still need it if we use self-signed certificates or k8s CA. It's safer to leave it there.

Reply:
wfm.